cmd/sink/main.go

```diff
@@ -141,6 +147,8 @@ func (sink *Sink) Receive(ctx context.Context, event cloudevents.Event) {
 
 func main() {
 	logger := log.New(os.Stderr, "", log.LstdFlags|log.Lmicroseconds|log.LUTC)
+	logger.Printf("Starting KubeArchive Sink with version '%s', commit SHA '%s', built at '%s'", version, commit, date)
```

Comment:
"built at" would indicate a location, for a date I think this should be just "built".

```suggestion
	logger.Printf("Starting KubeArchive Sink with version '%s', commit SHA '%s', built '%s'", version, commit, date)
```

Reply:
Changed!